superset-frontend/src/explore/components/controls/CheckboxControl.jsx

```diff
@@ -65,6 +65,7 @@ export default class CheckboxControl extends React.Component {
           <ControlHeader
             {...this.props}
             leftNode={this.renderCheckbox()}
+            role="button"
```

Comment:
This prop is redundant and https://github.com/apache/superset/pull/26602/files#diff-1786e2046531b3067d8ed91311079f60a31e6503a457a043c92c17f29ec31adaR114 already covered